src/js/components/Diagram/Diagram.js

```diff
@@ -250,6 +299,49 @@ const Diagram = forwardRef(({ connections, ...rest }, ref) => {
       connections={paths}
       {...rest}
     >
+      <defs>
+        <marker
+          id="filledTwo"
+          markerWidth="10"
+          markerHeight="10"
+          refX="2"
+          refY="6"
```

Comment:
Can we add a few words on what do these magic numbers symbolize? Also, it might be more maintainable to extract them to a const variable and reuse the const across the page.